tools/autodeployment/src/sibling-checker/index.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-unused-expressions */
```

Comment:
Are these really necessary?  Can we not change your code to fit our linting style?

Reply:
Regarding the `/* eslint-disable no-unused-expressions */` it'solved. 

But regarding `/* eslint-disable no-unused-expressions */`
I have tried until I run out of ideas.

Because the `firstCheck` calls the `secondCheck` and vice versa. 

```js
const a = () => {
b(); -> LINTER: "b" was used before it was defined.
}
const b = () => {
a()
}
```

Any idea about this is welcome.